PlayCover/Model/PlayApp.swift

```diff
@@ -138,6 +148,28 @@ class PlayApp: BaseApp {
         Uninstaller.clearExternalCache(info.bundleIdentifier)
     }
 
+    func createAlias() {
+        do {
+            try FileManager.default.createDirectory(atPath: PlayApp.aliasDirectory.path,
+                                                    withIntermediateDirectories: true,
+                                                    attributes: nil)
+        } catch {
+            Log.shared.log("Error creating app alias directory: \(error)")
```

Comment:
These error should be localised or just have `Log.shared.log` output `error` itself. If the error is just going to be logged by itself, it should be combined with the do/catch statement below.

Reply:
Done.